packages/web-frontend/src/containers/MapOverlayBar/Control.js

```diff
@@ -30,7 +42,6 @@ const Container = styled.div`
 
 const Header = styled.div`
```

Comment:
```suggestion
const Header = styled(FlexSpaceBetween)`
```
There are some flexbox utility components in flexbox.js which can be extended to save duplication.